crs_application/include/crs_application/common/config.h

```diff
@@ -105,6 +104,7 @@ struct PartRegistrationConfig
   std::string toolpath_file;
   std::array<double, 6> seed_pose;
   std::array<double, 6> simulation_pose;
+  double waypoint_edge_buffer;
```

Comment:
Shouldn't this be a parameter specific to motion planning?

Reply:
It's here because this is where the toolpath is loaded and it is immediately applied to the loaded toolpath